packages/esm-commons-lib/src/utils/summary-card-config-builder.ts

```diff
@@ -9,12 +11,39 @@ export const getSummaryCardProps = (schemaConfig, config = null) => {
     header: column.title,
     concept: column.concept,
     encounterTypes: column.encounterTypes,
-    getObsValue: async ([encounter]) => {
+    getObsValue: async (encounter) => {
       if (column.isConditionalConcept) {
-        return getConditionalConceptValue(encounter, column.conditionalConceptMappings, column.isDate);
+        return getConditionalConceptValue(encounter, column.conditionalconditionalConceptMappings, column.isDate);
       }
+
+      if (column.id == 'artInitiation' && column.encounterTypes?.length > 0) {
```

Comment:
We agreed not to use name specific conditions like art, because we want some of these functions to be reused, you might wanna do something like what I did for ART date and Reason.